x-pack/plugins/cloud_security_posture/public/pages/findings/findings_by_resource_table_pagination.tsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import {
+  EuiButtonEmpty,
+  EuiButtonIcon,
+  EuiContextMenuItem,
+  EuiContextMenuPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPopover,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import * as TEST_SUBJECTS from './test_subjects';
+
+interface Props {
+  pageSize: number;
+  options: number[];
+  hasNextPage: boolean;
+  hasPrevPage: boolean;
+  fetchNext(): void;
+  fetchPrev(): void;
+  setPageSize(size: number): void;
+}
+
+export const FindingsByResourceTablePagination = ({
```

Comment:
this custom pagination component includes 2 buttons: `prev` and `next`

tried to make it look like [EuiPagination](https://elastic.github.io/eui/#/navigation/pagination) by looking into the source and making sure this component is roughly the same, minus the parts we don't need.